src/loaders/OmeZarrLoader.ts

```diff
@@ -302,12 +383,17 @@ class OMEZarrLoader extends ThreadableVolumeLoader {
     return Promise.resolve({ imageInfo: imgdata, loadSpec: fullExtentLoadSpec });
   }
 
-  private async prefetchChunk(basePath: string, coords: TCZYX<number>, subscriber: SubscriberId): Promise<void> {
-    const separator = basePath.endsWith("/") ? "" : "/";
-    const key = basePath + separator + this.orderByDimension(coords).join("/");
+  private async prefetchChunk(
+    scaleLevel: NumericZarrArray,
+    coords: TCZYX<number>,
+    subscriber: SubscriberId
+  ): Promise<void> {
+    const { store, path } = scaleLevel;
+    const separator = path.endsWith("/") ? "" : "/";
+    const key = path + separator + this.orderByDimension(coords).join("/");
```

Comment:
in my experience, this sort of string manipulation is good to factor out and have unit tests for because it's an easy place to have unintentional results